amy/settings.py

```diff
@@ -19,17 +19,16 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = json.loads(os.environ.get('DEBUG', 'true'))
-# For deployment in production: 
-# DEBUG=false SECRET_KEY="..." ./manage.py runserver ...
+# SECURITY WARNING: don't run with DEBUG turned on in production!
+DEBUG = json.loads(os.environ.get('AMY_DEBUG', 'true'))
```

Comment:
On 2015-03-01 8:28 AM, Piotr Banaszkiewicz wrote:
> > +DEBUG = json.loads(os.environ.get('AMY_DEBUG', 'true'))
> It's there to handle 'true' / 'false' strings, ie. convert them to 
> True/False objects in Python.

That seems like a really big hammer for a really small nail - it would 
be a lot simpler, and more conformant with Unix convention, to simply 
require AMY_DEBUG=1.
Thanks
Greg